aion/config/prod.exs

```diff
@@ -1,61 +1,18 @@
 use Mix.Config
 
-# For production, we configure the host to read the PORT
-# from the system environment. Therefore, you will need
-# to set PORT=80 before running your server.
-#
-# You should also configure the url host to something
-# meaningful, we use this information when generating URLs.
-#
-# Finally, we also include the path to a manifest
-# containing the digested version of static files. This
-# manifest is generated by the mix phoenix.digest task
-# which you typically run after static files are built.
 config :aion, Aion.Endpoint,
-  http: [port: {:system, "PORT"}],
-  url: [host: "example.com", port: 80],
+  http: [port: 80],
+  url: [host: "aion.mrapacz.pl", port: 80],
```

Comment:
That could possibly by env var in future?

Reply:
Yes, in the future. So far I guess we can hardcode it as we're only deploying our app on one server. Since we are deleting heroku from .travis.yml, we can also hardcode it here and we'll make it environment variable the moment we add heroku as our staging platform.